Patch 16 (#11808)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/m365-dr-overview.md

There are three methods for ensuring that the _Tenant_ data location for a particular service does not change.

> 1. Product Terms: Exchange Online, SharePoint Online, OneDrive for Business and Microsoft Teams provisioned in Australia, Brazil, Canada, France, Germany, India, Japan, Qatar, South Korea, Norway, South Africa, Sweden, Switzerland, United Arab Emirates, United Kingdom, European Union and the United States have a commitment for customer data residency expressed in the [Product Terms](https://www.microsoft.com/licensing/terms/product/PrivacyandSecurityTerms/all).  For more information see the [Product Terms Data Residency page](m365-dr-product-terms-dr.md).

Reviewer: ```suggestion
1. Product Terms: Exchange Online, SharePoint Online, OneDrive for Business, and Microsoft Teams provisioned in Australia, Brazil, Canada, France, Germany, India, Japan, Qatar, South Korea, Norway, South Africa, Sweden, Switzerland, United Arab Emirates, United Kingdom, European Union, and the United States have a commitment for customer data residency expressed in the [Product Terms](https://www.microsoft.com/licensing/terms/product/PrivacyandSecurityTerms/all).  For more information see the [Product Terms Data Residency page](m365-dr-product-terms-dr.md).
```

| Viva Topics <br/> |-  <br/> |-  <br/> |X<sup>3</sup>  <br/> |
| Microsoft Purview <br/> |-  <br/> |-  <br/> |X<sup>3</sup>  <br/> |

> 1. Only available in the following countries/region: Australia, Brazil, Canada, France, Germany, India, Japan, Poland, Qatar, South Korea, Norway, South Africa, Sweden, Switzerland, United Arab Emirates, United Kingdom, European Union and the United States.

Reviewer: ```suggestion
1. Only available in the following countries/regions: Australia, Brazil, Canada, France, Germany, India, Japan, Poland, Qatar, South Korea, Norway, South Africa, Sweden, Switzerland, United Arab Emirates, United Kingdom, European Union, and the United States.
```

> 1. Available in _Local Region Geography_, _Expanded Local Region Geography_ (when the future data center is launched) and _Regional Geography countries/regions_

Reviewer: ```suggestion
1. Available in _Local Region Geography_, _Expanded Local Region Geography_ (when the future data center is launched), and _Regional Geography countries/regions_.
```

> 1. Only available for _Local Region Geography_ and _Expanded Local Region Geography_ (when the future data center is launced) countries.

Reviewer: ```suggestion
1. Only available for _Local Region Geography_ and _Expanded Local Region Geography countrise_ (when the future data center is launched).
```

|**Regional Geographies** |**Locations where customer data may be stored**  |
|---------|---------|

> |Macro Region Geography 1 - EMEA (Europe, Middle East and Africa) |  Austria, Finland, France, Ireland, Netherlands, Poland, Sweden  |

Reviewer: ```suggestion
|Macro Region Geography 1 - EMEA (Europe, Middle East, and Africa) |  Austria, Finland, France, Ireland, Netherlands, Poland, Sweden  |
```

|Australia   |Sydney, Melbourne   |
|Brazil   |Rio, Campinas   |
|Canada      |Quebec City, Toronto    |

> |European Union      |Austria (Vienna), Finland (Helsinki), France (Paris, Marseille), Ireland (Dublin), Netherlands (Amsterdam), Poland (Warsaw), (Sweden (Gävle, Sandviken, Staffanstorp)     |

Reviewer: ```suggestion
|European Union      |Austria (Vienna), Finland (Helsinki), France (Paris, Marseille), Ireland (Dublin), Netherlands (Amsterdam), Poland (Warsaw), Sweden (Gävle, Sandviken, Staffanstorp)     |
```
